### PR TITLE
doc: Add manual page generation for bitcoin-util

### PR DESCRIPTION
**Pull request by laanwj**
- Add `-version` option to `bitcoin-util`
- Add `bitcoin-util` call to `gen-manpages.sh`
- Add stub manual page `bitcoin-util.1`
- Add install of `bitcoin-util.1` to build system

### CODE REVIEW

File: doc/man/bitcoin-util.1 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.TH BITCOIN-UTIL "1"
+.SH NAME
+bitcoin-util \- manual page for bitcoin-util
+
+This is a placefolder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```
Comment (adamjonas) [inline doc/man/bitcoin-util.1:5]:
```suggestion
This is a placeholder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```

Comment (laanwj) [reply doc/man/bitcoin-util.1:5]: Thanks. We'll want to change this in the other files as well.

**Comment by laanwj** [comment]
Added scripted-diff commit to fix the typo in the stub manual pages.

**Review by fanquake — APPROVED** [review]
ACK bc99ae77e431740c0eda8a93152ce15600b3852a

```bash
src/bitcoin-util -version            
Bitcoin Core bitcoin-util utility version v21.99.0-bc99ae77e431
```

**Pull request merged**